scripts/ci/jenkins_pipeline_csi

```diff
@@ -25,9 +33,47 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('Build docker images') {
+            steps {
+                script {
+                    sh 'bash ./scripts/ci/run_variables_verification.sh'
+                    branch = sh(script:"echo $GIT_BRANCH | sed 's|/|.|g'",returnStdout:true).trim()
+                    specific_tag = "${IMAGE_VERSION}_b${BUILD_NUMBER}_${branch}"
+                    if (latest_tag.contains(GIT_BRANCH)) {
+                        controller_image_latest = docker.build("${CSI_CONTROLLER_IMAGE}", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_controller} .")
+                        node_image_latest = docker.build("${DOCKER_REGISTRY}/${CSI_NODE_IMAGE}", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_node} .")
+                    }
+                    controller_image_with_generated_tag = docker.build("${DOCKER_REGISTRY}/${CSI_CONTROLLER_IMAGE}:$specific_tag", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_controller} .")
+                    node_image_with_generated_tag = docker.build("${DOCKER_REGISTRY}/${CSI_NODE_IMAGE}:$specific_tag", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_node} .")
```

Comment:
in the original code, if the node build would fail, we would at least have the controller image available to pull.
why should we change that?

Reply:
in the original code, if the node build fails you won't know about it, the pipeline can finish in succeded mode and still don't push the node image.

Reply:
1. I disagree - did you test your statement?
2. I don't see how this is related to this conversation.

- if you think my statement is wrong, please elaborate.
- if you agree with my statement, please answer "why should we change that?".

Reply:
I was wrong, I work now with the original bash file